src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -276,14 +276,13 @@ private function fileExcerpt(string $file, int $line, int $srcContext = 3): stri
                 return "<span $m[1]>".str_replace('<br />', "</span><br /><span $m[1]>", $m[2]).'</span>';
             }, $code);
             $content = explode('<br />', $code);
-
             $lines = [];
             if (0 > $srcContext) {
                 $srcContext = \count($content);
             }
 
             for ($i = max($line - $srcContext, 1), $max = min($line + $srcContext, \count($content)); $i <= $max; ++$i) {
-                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><a class="anchor" name="line'.$i.'"></a><code>'.$this->fixCodeMarkup($content[$i - 1]).'</code></li>';
+                $lines[] = '<li'.($i == $line ? ' class="selected"' : '').'><code>'.$this->fixCodeMarkup($content[$i - 1]).'</code></li>';
```

Comment:
the FileLinkFormatter relies on the anchor link :)

https://github.com/symfony/symfony/blob/494ef421c554a78b38c6779c4b7deb9a20d89923/src/Symfony/Bundle/WebProfilerBundle/Resources/config/profiler.php#L72

Reply:
then we might need an option to put the ids on that page only (where there is a single file excerpt)

Reply:
try a longer line no. ;) the missing anchor here will break the url, in terms of the anchor wont be scrolled to.

Reply:
@ro0NL I think `/_profiler/open` does not use the ErrorRenderer to render file excerpt

Reply:
im looking at the same file twice -.-

the impact on the twig bridge's code extension is maybe even more severe :)

Reply:
How was this resolved?

Reply:
As written above, `FileLinkFormatter` relies on anchor links from `CodeExtension`, not `ErrorRenderer`.
Another feedback was made about `CodeExtension`.

On my side I searched for every occurrence of the line id / name and did not find anywhere were the ones from `ErrorRenderer` were used, and seeing that they were mostly unusable and could break at any time, I deduced that removing them was for the best.

Reply:
@Nyholm the ID anchor is preserved in `CodeExtension.php` :)  it's not needed in `HtmlErrorRenderer.php`

Reply:
Thank you for your explanations. I didnt understand it at first. 

👍